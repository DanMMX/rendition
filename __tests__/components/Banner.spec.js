import React from 'react';
import renderer from 'react-test-renderer';
import Provider from '../../src/components/Provider'
import Banner from '../../src/components/Banner'

test('Button renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Banner color='white'>
        <h1>Resin.io</h1>
        <p>
          Resin.io brings the benefits of Linux containers to the IoT. Develop
          iteratively, deploy safely, and manage at scale.
        </p>
      </Banner>
    </Provider>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
