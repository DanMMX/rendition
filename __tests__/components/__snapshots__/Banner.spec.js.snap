// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<div
  className="sc-bdVaJa jKopLr"
>
  <div
    className="sc-htoDjs kpLPJf sc-gzVnrw hnoozl"
    color="white"
  >
    <h1>
      Resin.io
    </h1>
    <p>
      Resin.io brings the benefits of Linux containers to the IoT. Develop iteratively, deploy safely, and manage at scale.
    </p>
  </div>
</div>
`;
